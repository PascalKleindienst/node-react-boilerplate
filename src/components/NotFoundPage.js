// ==============================================
// NOT FOUND PAGE COMPONENT =====================
// ==============================================
import React from 'react';
import { Link } from 'react-router-dom';

const NotFoundPage = () => (
    <div>
        404! - <Link to="/">Go home, you are drunk!</Link>
    </div>
);

export default NotFoundPage;
