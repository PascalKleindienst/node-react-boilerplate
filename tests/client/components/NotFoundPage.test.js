// ==============================================
// NOT FOUND PAGE TEST ==========================
// ==============================================
import React from 'react';
import { shallow } from 'enzyme';
import NotFoundPage from '../../../src/components/NotFoundPage';

describe('my NotFoundPage component', () => {
    test('should render correctly', () => {
        const wrapper = shallow(<NotFoundPage />);
        expect(wrapper).toMatchSnapshot();
    });
});
