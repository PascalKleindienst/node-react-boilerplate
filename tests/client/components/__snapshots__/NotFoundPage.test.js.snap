// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`my NotFoundPage component should render correctly 1`] = `
<div>
  404! - 
  <Link
    replace={false}
    to="/"
  >
    Go home, you are drunk!
  </Link>
</div>
`;
